src/cmd/flux-exec.c

```diff
@@ -182,10 +190,26 @@ static void stdin_cb (flux_reactor_t *r, flux_watcher_t *w,
 static void signal_cb (int signum)
 {
     flux_subprocess_t *p = zlist_first (subprocesses);
+    struct timeval now;
+
+    if (signum == SIGINT) {
+        struct timeval diff;
+
+        gettimeofday (&now, NULL);
+
+        timersub (&now, &last, &diff);
+
```

Comment:
Our `monotime` interface (`libutil/monotime.h`) or at least `clock_gettime`  should be used here instead of `gettimeofday`, which is obsolesced by POSIX I think. See [here](http://pubs.opengroup.org/onlinepubs/9699919799/functions/gettimeofday.html)

Reply:
ahh wasn't aware, updated & repushed